### DISCUSSION TITLE
'VirtualMachine' object has no attribute 'oob_ip_id'

### DISCUSSION THREAD

**Discussion by krasotinpa** [netbox-community/netbox#18231] [Help Wanted!]
Hi - 

After updating Netbox to the version 4.1.8 noticed problem with editing IP address assigned to VM's interface.
Pressing 'Edit' button leads to:
```
<class 'AttributeError'>

'VirtualMachine' object has no attribute 'oob_ip_id'

Python version: 3.12.8
NetBox version: 4.1.8
Plugins: None installed
```
I've already create [bug report](https://github.com/netbox-community/netbox/issues/18230). But may be problem is in my installation?

Other IPs are edited correctly

**Comment by candlerb** [comment]
Not your installation, it's a bug. I can reproduce it on demo.netbox.dev:

- add address 1.2.3.4/32 to vm101 interface eth0
- try to edit it: https://demo.netbox.dev/ipam/ip-addresses/186/edit/
